src/services/JWTService.ts

```diff
@@ -15,42 +16,36 @@ class JWTService {
     const decodedToken: any = JWT.decode(token, {complete: true});
 
     // Check if config is valid
-    if (!config || !config.azure || !config.azure.tennant || !config.azure.appId || !config.azure.issuer || !config.azure.jwk_endpoint) {
-      throw new AuthorizationError(ERRORMESSAGES.AZURE_CONFIGURATION_NOT_VALID);
-    }
+    if (!config || !config.azure || !config.azure.tennant || !config.azure.appId || !config.azure.issuer || !config.azure.jwk_endpoint) { throw new AuthorizationError(ERRORMESSAGES.AZURE_CONFIGURATION_NOT_VALID); }
```

Comment:
There was no need to change this to a single line and makes it less readable.

Reply:
agree, will revert, didn't mean to, we can probably add prettier so it formats it well and add it as precommit hook